tf_agents/cogment_verse_tf_agents/reinforce/reinforce.py

```diff
@@ -30,48 +31,47 @@ class ReinforceAgent:
     # pylint: disable=too-many-instance-attributes
     def __init__(
         self,
-        id,
-        obs_dim,
-        act_dim,
-        epsilon=0.01,
-        gamma=0.95,
+        id="*",
```

Comment:
is there actually any usage of the id?

If it's not useful just get rid of it. Otherwise "*" doesn't feel like a good default. Maybe no default?